lib/algos/list/MPU.js

```diff
@@ -34,6 +35,28 @@ class MultipartUploads {
         this.logger = logger;
     }
 
+    genMDParams() {
+        const params = {};
+        if (this.params.keyMarker) {
+            params.gt = `overview${this.params.splitter}` +
+                `${this.params.keyMarker}${this.params.splitter}`;
+            if (this.params.uploadIdMarker) {
+                params.gt += `${this.params.uploadIdMarker}`;
+            }
+            // advance so that lower bound does not include the supplied
+            // markers
+            params.gt = inc(params.gt);
```

Comment:
I've got them all from the old code https://github.com/scality/MetaData/pull/1009/files#diff-b596dc6bd6d51b111a52033729463bd1L189. Maybe @LaurenSpiegel knows what it does.